index.html

```diff
@@ -290,6 +293,33 @@ <h3>Contact me</h3>
         </form>
       </div>
     </section>
-    <script src="script.js"></script>
+    
+    <!-- popup code starts -->
+    <div class="popUp">
+      <div class="popHead">
+          <div class="popn">
+            <h2 class="popProj">h2</h2>
+            <button class="close-button" type="submit"> &times;</button>
+          </div>
+          <div class="popName"></div>
+      </div>
+      <div class="popPic">
+          <img class="popPic1" src="images/" alt="">
+      </div>
+      <div class="popDet">
+          <div class="popTex"> </div>
+          <div class="popSocial">
+              <ul class="popList" id="popList1">
+                  <li class="pplist"></li>
+              </ul>
+              <div class="live">
+                <a id="popUpLive" class="popUpBtn" href="#">See Live &nbsp; &nbsp; &nbsp;<img src="photo/ben/Icon-Export.png" alt="live"></a>
+                <a id="popUpSource" class="popUpBtn" href="#">See Source  &nbsp; &nbsp;<img src="photo/ben/Icon-GitHub11.png" alt="git"></a>
+              </div>
+          </div>
+      </div>
+    </div>
```

Comment:
- [ ] As the previous reviewer mentioned, kindly create the popup and populate it from the data created in an array using JavaScript, this will make the popup generate for every project differently with the right data
the array can be like this:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```